Makefile

```diff
@@ -16,24 +14,29 @@ help:
 	${MAKE} build
 
 build:
-	-rm .docker-build
-	${DOCKERCOMPOSE} -f docker-compose.yml build irc
+	docker build -t standup_irc .
 	touch .docker-build
 
 run: .docker-build
-	${DOCKERCOMPOSE} up irc
+	docker run --env-file .env_dev -v "$PWD:/app" standup_irc
```

Comment:
I read through the code and the pg stuff is optional. If it's there, then standup-irc will persist things like joined channels. If not, no big deal.

That covers my thoughts.